### PR TITLE
feat:category page

### PR DESCRIPTION
**Pull request by deepak29-git**
When you click on any one of the category then it will be redirected with the selected category

### CODE REVIEW

**Review by Mohitkumar33 — APPROVED** [review]

File: src/Pages/SelectedCategory/SelectedCategory.jsx [hunk 1]
```diff
@@ -1,47 +1,108 @@
...
+            return (
+              <div key={_id} className="video-card">
+                <Link
+                  to={`/watch/${_id}`}
+                  onClick={() => addToHistory(video, setHistory)}
+                >
+                  <img className="thumbnail-image" src={image} alt={title} />
+                </Link>
+                <div>
+                  {/* <div className="icon-container"> */}
```
Comment (tushargoyal812) [inline src/Pages/SelectedCategory/SelectedCategory.jsx:65]: remove comments

**Review by tushargoyal812 — COMMENTED** [review]
variable naming is very good.
liked how you structured folders.
code is very clean and to the point.good work

File: src/Pages/SelectedCategory/SelectedCategory.jsx [hunk 2]
```diff
@@ -1,47 +1,108 @@
...
+import { Sidebar } from "../../components/Sidebar/Sidebar";
+import { Modal } from "../../components/Modal/Modal";
+import { addToHistory } from "../../Utility/add-to-history";
+import { useHistory } from "../../Context/history-context";
+import { usePlaylist } from "../../Context/platlist-context";
+import { useWatchLater } from "../../Context/watch-leter-context";
+import { deleteFromWatchLater } from "../../Utility/deleteFromWatchLater";
+import { addToWatchLater } from "../../Utility/addToWatchLater";
+import { useAuth } from "../../Context/auth-context";
 export const SelectedCategory = () => {
```
Comment (ajay-mandaviya) [inline src/Pages/SelectedCategory/SelectedCategory.jsx:18]: if you have more than two or three file into one folder make index.js file import all file her and export it here 

and import all this file from your index.js file 
so it's looks clean

File: src/Pages/SelectedCategory/SelectedCategory.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  box-shadow: 0px 0px 5px var(--grey-color);
+}
+
+.video-card:hover {
+  box-shadow: 0px 0px 15px var(--black-color);
+}
+
+.playlist-btn{
+    background-color: var(--grey-shade-1);
+    border-radius: 0;
```
Comment (tanishk15sharma) [inline src/Pages/SelectedCategory/SelectedCategory.css:15]: you can remove this  `border-radius`.

File: src/backend/db/videos.js [hunk 4]
```diff
@@ -44,130 +47,90 @@ export const videos = [
...
-  //   creator: "Sentdex",
-  //   description:
-  //     "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
-  // },
-  // {
-  //   _id:"9kqWIuAlFAI",
-  //   title: "Dil sambhal ja jara & Iss dard e dil ki by Arunita & Mohammad Irfan at Grand Finale",
-  //   creator: "Sentdex",
-  //   description:
-  //     "At vero eos et accusamus et iusto odio dignissimos ducimus qui blanditiis praesentium voluptatum deleniti atque corrupti quos dolores et quas molestias excepturi sint occaecati cupiditate non provident, similique sunt in culpa qui officia deserunt mollitia animi, id est laborum et dolorum fuga.",
```
Comment (ajay-mandaviya) [inline src/backend/db/videos.js:170]: once you complete with project make sure delete it comments

File: src/Pages/SelectedCategory/SelectedCategory.jsx [hunk 5]
```diff
@@ -1,47 +1,108 @@
...
-          </div>
+    <div style={{ backgroundColor: bg }}>
+      <Header />
+
+      <div className="grid-container">
+        <Sidebar />
+        <div className="postion-center">{modal && <Modal />}</div>
+        {loader && <h1 className="center">...Loading</h1>}
+        <div className="video-listing-main ">
+          {filteredCategoies.map((video) => {
```
Comment (tanishk15sharma) [inline src/Pages/SelectedCategory/SelectedCategory.jsx:54]: Suggestion : You can create another component of `FilteredCategoriesVideo` and pass the component in the map , and send mapped video as `props` to the component by this your code will look short and refactored

**Review by ajay-mandaviya — APPROVED** [review]

**Review by tanishk15sharma — COMMENTED** [review]
I have suggested some changes to you and you can also make a `index.js` file where export all your similar files . as you are importing too many files in `SelectedCategory.jsx`.

File: src/Pages/SelectedCategory/SelectedCategory.jsx (hunk 5 above)
Comment (deepak29-git) [reply src/Pages/SelectedCategory/SelectedCategory.jsx:54]: yes I will do that

File: src/Pages/SelectedCategory/SelectedCategory.css (hunk 3 above)
Comment (deepak29-git) [reply src/Pages/SelectedCategory/SelectedCategory.css:15]: ok

**Pull request merged**